web-api/src/models.rs

```diff
@@ -160,8 +160,7 @@ pub(crate) struct IngestedDocument {
     pub(crate) properties: DocumentProperties,
 
     /// The tags associated to the document.
-    #[serde(default, deserialize_with = "deserialize_empty_option_string_as_none")]
-    pub(crate) tags: Option<String>,
+    pub(crate) tags: Vec<String>,
```

Comment:
:pushpin: We could reject 'tags` that are empty strings or invalid id. This is not exposed in the specification so we have a bit of time to finalize this.

Reply:
i'll add a follow up ticket to the backlog.